js/src/Map.js

```diff
@@ -302,6 +302,11 @@ var LeafletMapView = utils.LeafletDOMWidgetView.extend({
         LeafletMapView.__super__.processPhosphorMessage.apply(this, arguments);
         switch (msg.type) {
             case 'resize':
+                var container = this.map_container;
+                if (container.clientHeight !== container.parentElement.clientHeight) {
+                    container.style.height = container.parentElement.clientHeight + 'px';
+                }
+
```

Comment:
I'm not sure it's the right fix. I guess it would be cleaner to fix it in full CSS, which would mean making the `map_container` fill the widget container?

Reply:
Maybe something like this fix #352? I basically replicate the "old" behavior, the one that you see on the last release. I fear that your fix only works on JupyterLab, and that it does not take into account the `map.layout` attribute.

Reply:
CSS still scares me, so yeah, this is probably not a good solution (but it did work in the notebook as well as lab), I removed this from this PR.